.prettierrc

```diff
@@ -1,4 +1,6 @@
 {
   "singleQuote": true,
-  "trailingComma": "all"
-}
\ No newline at end of file
+  "trailingComma": "all",
+  "tabWidth": 2,
+  "printWidth": 100
+}
```

Comment:
Do you think we might need this?
"endOfLine": "auto"
The default value of endOfLine is "lf" based on the document. It is mainly for Linux and Macos users.

https://prettier.io/docs/en/options.html#end-of-line

Reply:
yep, thanks for reminding me,  I will add it soon

Reply:
And prettier please. We really need that in the pre-commit script.

Reply:
> And prettier please. We really need that in the pre-commit script.

If I'm not wrong. The command "format" is the one for prettier and it's already there.

Reply:
Yes, I just notice that and I have deleted the comment.